src/database/sql/audio.py

```diff
@@ -324,7 +352,8 @@ def get_relaxed(self, audio_id: str) -> str:
         query = """
                 SELECT relaxed, relaxed_confidence, last_updated
                 FROM Audio
-                WHERE audio_release=:rel AND audio_side=:side AND audio_track=:track
+                WHERE audio_release=:rel AND audio_side=:side
+                AND audio_track=:track
```

Comment:
If you want this AND `audio_track=:track` to be on a new line, please indent it.